demo/site/server.ts

```diff
@@ -35,33 +35,6 @@ app.prepare().then(() => {
                 }
             }
 
-            let maxAge: string | undefined;
```

Comment:
Is this ok to take these changes from starter?

Reply:
no, we should keep this. the `setHeader` override should return `this` instead of undefined for the ignore part.